graphql_compiler/compiler/filters.py

```diff
@@ -321,19 +341,22 @@ def _process_name_or_alias_filter_directive(filter_operation_info, location, con
     alias_field_inner_type = strip_non_null_from_type(alias_field_type.of_type)
     if alias_field_inner_type != name_field_type:
         raise GraphQLCompilationError(u'Cannot apply "name_or_alias" to type {} because the '
-                                      u'"name" field and the inner type of the "alias" field '
-                                      u'do not match: {} vs {}'.format(filtered_field_type,
-                                                                       name_field_type,
-                                                                       alias_field_inner_type))
+                                      u'"{}" field and the inner type of the "{}" field '
+                                      u'do not match: {} vs {}'
+                                      .format(filtered_field_type, name_field_name,
+                                              alias_field_name, name_field_type,
+                                              alias_field_inner_type))
 
     argument_inferred_type = name_field_type
     argument_expression, non_existence_expression = _represent_argument(
         location, context, parameters[0], argument_inferred_type)
 
     check_against_name = expressions.BinaryComposition(
-        u'=', expressions.LocalField('name'), argument_expression)
+        u'=', expressions.LocalField(name_field_name, name_field.type), argument_expression)
```

Comment:
name_field_name seems to be a constant. Why not uppercase it to make that clear?

Reply:
It's only a function-level constant, not a module-level one. The Google Python Style Guide says that only module-level constants are capitalized, and all function-level values (constant or variable) are lowercase: http://google.github.io/styleguide/pyguide.html#316-naming